colors/seoul256.vim

```diff
@@ -34,106 +34,87 @@
-        \ 106: '#87af00', 107: '#87af5f', 108: '#87af87',
-        \ 109: '#87afaf', 110: '#87afd7', 111: '#87afff',
-        \ 112: '#87d700', 113: '#87d75f', 114: '#87d787',
-        \ 115: '#87d7af', 116: '#87d7d7', 117: '#87d7ff',
-        \ 118: '#87ff00', 119: '#87ff5f', 120: '#87ff87',
-        \ 121: '#87ffaf', 122: '#87ffd7', 123: '#87ffff',
-        \ 124: '#af0000', 125: '#af005f', 126: '#af0087',
-        \ 127: '#af00af', 128: '#af00d7', 129: '#af00ff',
-        \ 130: '#af5f00', 131: '#af5f5f', 132: '#af5f87',
-        \ 133: '#af5faf', 134: '#af5fd7', 135: '#af5fff',
-        \ 136: '#af8700', 137: '#af875f', 138: '#af8787',
-        \ 139: '#af87af', 140: '#af87d7', 141: '#af87ff',
-        \ 142: '#afaf00', 143: '#afaf5f', 144: '#afaf87',
-        \ 145: '#afafaf', 146: '#afafd7', 147: '#afafff',
-        \ 148: '#afd700', 149: '#afd75f', 150: '#afd787',
-        \ 151: '#afd7af', 152: '#afd7d7', 153: '#afd7ff',
-        \ 154: '#afff00', 155: '#afff5f', 156: '#afff87',
-        \ 157: '#afffaf', 158: '#afffd7', 159: '#afffff',
-        \ 160: '#d70000', 161: '#d7005f', 162: '#d70087',
-        \ 163: '#d700af', 164: '#d700d7', 165: '#d700ff',
-        \ 166: '#d75f00', 167: '#d75f5f', 168: '#d75f87',
-        \ 169: '#d75faf', 170: '#d75fd7', 171: '#d75fff',
-        \ 172: '#d78700', 173: '#d7875f', 174: '#d78787',
-        \ 175: '#d787af', 176: '#d787d7', 177: '#d787ff',
-        \ 178: '#d7af00', 179: '#d7af5f', 180: '#d7af87',
-        \ 181: '#d7afaf', 182: '#d7afd7', 183: '#d7afff',
-        \ 184: '#d7d700', 185: '#d7d75f', 186: '#d7d787',
-        \ 187: '#d7d7af', 188: '#d7d7d7', 189: '#d7d7ff',
-        \ 190: '#d7ff00', 191: '#d7ff5f', 192: '#d7ff87',
-        \ 193: '#d7ffaf', 194: '#d7ffd7', 195: '#d7ffff',
-        \ 196: '#ff0000', 197: '#ff005f', 198: '#ff0087',
-        \ 199: '#ff00af', 200: '#ff00d7', 201: '#ff00ff',
-        \ 202: '#ff5f00', 203: '#ff5f5f', 204: '#ff5f87',
-        \ 205: '#ff5faf', 206: '#ff5fd7', 207: '#ff5fff',
-        \ 208: '#ff8700', 209: '#ff875f', 210: '#ff8787',
-        \ 211: '#ff87af', 212: '#ff87d7', 213: '#ff87ff',
-        \ 214: '#ffaf00', 215: '#ffaf5f', 216: '#ffaf87',
-        \ 217: '#ffafaf', 218: '#ffafd7', 219: '#ffafff',
-        \ 220: '#ffd700', 221: '#ffd75f', 222: '#ffd787',
-        \ 223: '#ffd7af', 224: '#ffd7d7', 225: '#ffd7ff',
-        \ 226: '#ffff00', 227: '#ffff5f', 228: '#ffff87',
-        \ 229: '#ffffaf', 230: '#ffffd7', 231: '#ffffff',
-        \ 232: '#080808', 233: '#121212', 234: '#1c1c1c',
-        \ 235: '#262626', 236: '#303030', 237: '#3a3a3a',
-        \ 238: '#444444', 239: '#4e4e4e', 240: '#585858',
-        \ 241: '#626262', 242: '#6c6c6c', 243: '#767676',
-        \ 244: '#808080', 245: '#8a8a8a', 246: '#949494',
-        \ 247: '#9e9e9e', 248: '#a8a8a8', 249: '#b2b2b2',
-        \ 250: '#bcbcbc', 251: '#c6c6c6', 252: '#d0d0d0',
-        \ 253: '#dadada', 254: '#e4e4e4', 255: '#eeeeee' }
-    else
-        let s:rgb_map =
```

Comment:
Funnily enough, if you take ALL of these values, convert each value into a float, interpret it as "Generic RGB", convert it to sRGB and back to hex, you get the colors in the table above.